ui/components/src/theme/index.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type {} from './types/ThemeExtension';
+import { PersesColor } from './palette';
+
+export * from './theme';
+
+//Use Typescript interface augmentation to extend the MUI type definition
```

Comment:
Awesome you got this working, I remember getting pretty frustrated with it awhile back